cmd/vcert/result_writer.go

```diff
@@ -195,6 +276,11 @@ func (r *Result) Flush() error {
 			if err != nil {
 				return fmt.Errorf("failed to encode pkcs12: %s", err)
 			}
+		} else if r.Config.Format == "jks" {
```

Comment:
I think as this value is being used in more than 2 places, we should convert it on a constant variable.

Reply:
done!